hack/check_license.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 Red Hat, Inc.
+#
+
+git remote add upstream https://github.com/kubevirt/kubevirt.git > /dev/null 2>&1
+git fetch upstream > /dev/null
```

Comment:
I do not think a target should fetch such things.
It will be better to add a list of paths to ignore.

Reply:
For CI, we don't need it, locally we do i think

anyhow if the go-header is better, then i will close this PR

thanks